sync_radar_data_globus.py

```
@@ -64,10 +49,18 @@
 # for automatic authentication
 TRANSFER_RT_FILENAME = HOME + "/.globus_transfer_rt"
 
-# UUID of your endpoint, retrieve from endpoint information at: https://www.globus.org/app/endpoints
-PERSONAL_UUID = ''
-# Client ID retrieved from https://auth.globus.org/v2/web/developers
-CLIENT_ID = ''
+# UUID of your endpoint, retrieve from endpoint info at: https://www.globus.org/app/endpoints
+# Or from the filesystem that globusconnectpersonal is installed on. Note that the following
+# line is for LINUX operating systems and may be different for MAC OS.
+# Note: this assumes you are running globusconnectpersonal & this script on the same filesystem
+PERSONAL_UUID_FILENAME = HOME + "/.globusonline/lta/client-id.txt"
+
+if isfile(PERSONAL_UUID_FILENAME):
```

REVIEW 15 and 39 isfile will return false then what will happen? maybe you should raise an error if the file doesn't exist. 

However, great use of with!

---

I changed this to raise a file not found error, that would have been a bug since PERSONAL_UUID is assumed to be set when it's used below.